docs/_posts/examples/3400-01-09-disable-rotation.html

```diff
@@ -0,0 +1,23 @@
+---
+layout: example
+category: example
+title: Disable Map Rotation
+description: Disable right-click map rotation
+---
+
+<div id='map'></div>
+<script>
+mapboxgl.accessToken = 'pk.eyJ1IjoibWFubnkiLCJhIjoiMzBCNHFQUSJ9.VQQ9ZSW7viFT1yLhLiWLSA';
```

Comment:
Leave this line out. The access token will be automatically inserted into the example code, and it will use the actual access token for the user viewing the page if they are logged in.